lib/presentation/resources/todo_cards/todo_cards_state.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:clean_architecture_demo/domain/model/todo_model.dart';
+import 'package:equatable/equatable.dart';
+
+class TodoCardsState extends Equatable {
+  final List<TodoModel>? todoCards;
+
+  final Error? error;
```

Comment:
@dilumdesilva here instead of having 2 variables we can use single Either class variable.

Reply:
I will leave this like this for now as it requires cubit and screen level changes.